test-suite/tests/frame-g010-out.jsonld

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": {
+    "@vocab": "http://example.com/",
+    "loves": {
+      "@type": "@id"
+    },
+    "unionOf": {
+      "@type": "@id",
+      "@id": "owl:unionOf",
+      "@container": "@list"
+    },
+    "Class": "owl:Class"
+  },
+  "@graph": [
+    {
+      "@graph": [
+        {
```

Comment:
The way the algorithm works, this graph object should come at the end of the array, as the node object values of the top-level `@graph` are processed in lexicographical order. The first entry gets a blank-node, assigned as `_:b0`, so the keys of the object map are `["ActTwo", "Capulet", "Montague", "Person", "_:b0"]`, when sorted, they have the same order.

Otherwise, this looks correct.

Reply:
Ah, good catch, i missed the intermediate`@id` insertion.

Reply:
@gkellogg pushed it now